plugins/backstage-plugin-env0/src/components/env0-environment-details-card/env0-environment-details-card.tsx

```diff
@@ -108,8 +108,11 @@ export const Env0EnvironmentDetailsCard = () => {
       <CardActions>
         <Button color="primary" variant="contained">
           <Link
-            to={getEnv0EnvironmentUrl({ environmentId: environment.id, projectId: environment.projectId })}
-            color="textPrimary"
+            to={getEnv0EnvironmentUrl({
+              environmentId: environment.id,
+              projectId: environment.projectId,
+            })}
+            color="inherit"
```

Comment:
why this change?

Reply:
it looked bad on light mode, 
<img width="150" alt="image" src="https://github.com/user-attachments/assets/563395ba-e04a-45fa-b272-da0bf00bcb54" /> vs 
<img width="150" alt="image" src="https://github.com/user-attachments/assets/0c95246a-ca4d-4e4b-8a8e-797774d19f83" />

Reply:
dark mode also looks good